dali/kernels/signal/moving_mean_square_test.cc

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include <vector>
+#include "dali/test/tensor_test_utils.h"
+#include "dali/kernels/signal/moving_mean_square.h"
+
+namespace dali {
+namespace kernels {
+namespace test {
+
+namespace {
+const int kNDims = 1;
+}
+
+
+template<class InputType>
+class MovingMeanSquareCpuTest : public ::testing::Test {
+ public:
+  MovingMeanSquareCpuTest() {
+    input_.resize(dataset_size(shape_));
```

Comment:
In my opinion, just using `volume(shape)` is more descriptive

Reply:
shape.num_elements()?